test.dm

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file name should be test.md not test.dm also, this file should be ignored in `.gitignore` just as you did that of node_modules.